### ISSUE TITLE
Include Discover in the "install the Software Flatpak Plugin" page

### ISSUE THREAD

**Issue by ohyrans** [flatpak/flatpak.github.io#259]
Currently there is only GNOME Software there (as an example https://flatpak.org/setup/Ubuntu/ [see the Flatpak Plugin section]) and as Discover also have a backend prepped for flatpak it would be grand to have that included with for example: "Or if you use KDE Plasma, the plugin for Discover Software Center" and then the code snippet underneath

**Comment by andreasn** [comment]
In Debian it's plasma-discover-backend-flatpak, according to https://wiki.debian.org/FlatpakHowto
I think it's the same in Ubuntu, but @ohyrans would need to verify.

**Comment by allanday** [comment]
The whole point of these setup pages is to keep the instructions to an absolute minimum, and avoid users having to read up on options. I think I'd prefer to keep the Ubuntu page focused on the default Ubuntu experience therefore.

It'd be fine to add a page for Kubuntu though.

**Comment by andreasn** [comment]
Yeah, Kubuntu is probably what most people run if they run Discover on a Ubuntu family distro.

**Comment by ohyrans** [comment]
The same issue exists on the Debian page, and I do think (I am not sure) that Mageia and Opensuse has Discover.

**Comment by ysteina** [comment]
Kubuntu 18.04 users can install the Flatpak backend directly from Settings in the KDE Software Store ('Discover'). Or from the terminal (for both Kubuntu 17.10 and 18.04): `sudo apt install plasma-discover-flatpak-backend`

**Comment by allanday** [comment]
Would someone like to create a branch with instructions for Kubuntu?

**Comment by allanday** [comment]
Draft based on the Ubuntu instructions:

```
- name: "Kubuntu"
  logo: "kubuntu.svg"
  info: >  
    <ol class="distrotut">
      <li>
        <h2>Install Flatpak</h2>
        <p>To install Flatpak on Kubuntu 18.10 (Cosmic Cuttlefish), simply run:</p>
        <pre><code>
          <span class="unselectable">$</span> sudo apt install flatpak
        </code></pre>        
        <p>With older Kubuntu versions, the official Flatpak PPA is the recommended way to install Flatpak. To install it, run the following in a terminal:</p>
        <pre><code>
          <span class="unselectable">$</span> sudo add-apt-repository ppa:alexlarsson/flatpak
          <span class="unselectable">$</span> sudo apt update
          <span class="unselectable">$</span> sudo apt install flatpak
        </code></pre>
      </li>
      <li>
        <h2>Install the Discover Flatpak backend</h2>
        <p>The Flatpak plugin for the Software app makes it possible to install apps without needing the command line. To install, run:</p>
        <pre><code>
          <span class="unselectable">$</span> sudo apt install plasma-discover-flatpak-backend
        </code></pre>
      </li>
      <li>
        <h2>Add the Flathub repository</h2>
        <p>Flathub is the best place to get Flatpak apps. To enable it, run:</p>
        <pre><code>
          <span class="unselectable">$</span> flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo
        </code></pre>
      </li>
      <li>
        <h2>Restart</h2>
        <p>To complete setup, restart your system. Now all you have to do is <a href="https://flathub.org/">install some apps</a>!</p>
      </li>
    </ol>
```

Can someone confirm that this is correct?

**Comment by ysteina** [comment]
Commands are correct, as tested on a Kubuntu 18.10 install. 
Maybe it is worth noting that the Discover Flatpack backend is available only for releases 18.04 and later (it isn't in the repos of 16.04 and 14.04 LTS). The backend can also be installed from Discover itself (in Settings under Missing Backends), but since you want to keep the instructions to a minimum it might be unnecessary.
